<!DOCTYPE HTML>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <title>Rohan Ghosh Dastidar</title>
  <meta name="author" content="Rohan Ghosh Dastidar">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <link rel="stylesheet" type="text/css" href="styles.css">
  <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>üåê</text></svg>">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css">
</head>

<body>
  <script src="script.js"></script>
  <div class="container">
    <header>
      <h1 class="name">Rohan Ghosh Dastidar</h1>

      <nav class="navbar">
        <ul>
          <li><a href="index.html#home">Home</a></li>
          <li><a href="index2.html#projects">Projects</a></li>
          <li><a href="index2.html#recent-news">Recent News</a></li>
          <li><a href="index2.html#contact">Contact</a></li>
        </ul>
      </nav>

    </header>

    <section class="about">
      <h2>About Me</h2>
      <div class="about-content">
        <div class="about-text">
          <p>
            Hi! I am Rohan, a fourth-year undergraduate in the
            <a href="https://www.iitkgp.ac.in/department/CH">Chemical Engineering</a> department at the
            <a href="https://www.iitkgp.ac.in/">Indian Institute of Technology, Kharagpur</a>, enrolled in its Dual Degree course.
            Additionally, I am pursuing a minor degree in
            <a href="http://www.bt.iitkgp.ac.in/">Biotechnology and Biochemical Engineering</a>. To expand my domain knowledge, I am also doing micro-specializations in High Performance Computing (<a href = "http://www.ccds.iitkgp.ac.in/">Centre for Computational & Data Sciences</a>) and in Microfluidics & Nano patterning (<a href = "https://www.iitkgp.ac.in/department/NT">School of Nanoscience & Technology</a>).
          </p>
          
          <p>
            My research interests lie broadly in Deep Learning and associated ML frameworks applied to Computational Protein Engineering, Mathematical modelling of biological systems, and Structural Bioinformatics.
          </p>
          
          <p class="contact">
              <p>
                  <i class="fa-solid fa-envelope"></i> <a>Institute Email:</a> rgdastidar2069@kgpian.iitkgp.ac.in <br>
                  <i class="fa-regular fa-envelope"></i> <a>Personal Email:</a> rgdastidar2069@gmail.com
              </p>
              
              <a href="https://www.linkedin.com/in/rohan-ghosh-dastidar-984577255/">
                  <i class="fa-brands fa-linkedin"></i> LinkedIn
              </a> &nbsp;|&nbsp;
              
              <a href="https://github.com/rohangd03">
                  <i class="fa-brands fa-github"></i> GitHub
              </a> &nbsp;|&nbsp;
              
              <a href="files/rohan_22ch30028_resume_2026_v4.pdf" target="_blank" class="cv-link">
                  <i class="fa-solid fa-file-pdf"></i> Resume 
              </a>
          </p>

        </div>

        <div class="profile-photo">
          <a href="images/andaman_profile_photo.jpeg">
            <img src="images/andaman_profile_photo.jpeg" alt="Rohan Ghosh Dastidar" class="photo">
          </a>
        </div>
      </div>

    </section>
    
<!-- ------------------------------------------------------------------------------------- -->

    <section class="research">
      <h2>Research Projects</h2>

            <!-- ----------- (5) BTP --------------------------------------------------------------- -->
      <div class="research-item">
        <a href="images/btp_1_graphical_abstract.avif">
          <img src="images/btp_1_graphical_abstract.avif" alt="Computational Prediction of disordered regions in proteins from DNA sequence" class="research-img">
        </a>

        <div class="research-description">
          <h3 class="papertitle">Computational Prediction of disordered regions in proteins from DNA sequence</h3>
          <p class = "justified-text">
            <i>Supervisor: Prof. Ranjit Prasad Bahadur, Dept. of Bioscience & Biotechnology, IIT Kharagpur, India</i> <br>
             In my Bachelor Thesis Project, I developed a bioinformatics pipeline to predict intrinsic disorder in proteins using machine learning. Furthermore, I performed genomic analysis of specific organisms from the three domains ‚Äì bacteria, archaea, eukarya to assess
             the nature of disordered regions in those proteomes.
          </p>
        </div>
      </div>


      <!-- ----------- (4) IISC BANGALORE --------------------------------------------------------------- -->
      <div class="research-item">
        <a href="images/csb_summer_intern_graphical_abstract.avif">
          <img src="images/csb_summer_intern_graphical_abstract.avif" alt="Stability of mRNA expression in toggle switch across varying noise levels" class="research-img">
        </a>

        <div class="research-description">
          <h3 class="papertitle">Investigation of the role of transcriptional noise in Epithelial-Mesenchymal state switching</h3>
          <p class = "justified-text">
            <i>Supervisor: Prof. Mohit Kumar Jolly, Cancer Systems Biology Lab, Param Hansa Centre for Computational Oncology, IISc Bengaluru, India</i> <br>
             In this project, I investigated the role of Transcriptional Noise in Epithelial-Mesenchymal Heterogeneity to study phenotypic switching. By simulating gene regulatory networks using Stochastic Differential Equations and high-throughput parameter
scans, I obtained time series data of mRNA expressions by different nodes in a genetic toggle switch and then performed statistical measurements on such expression patterns. I also implemented a ‚Äùpartitioned leaping‚Äù simulation algorithm from available literature to optimize space and time complexity of stochastic simulations instead of depending on conventional simulation schemes such as Gillespie or SDE's to mediate time efficiency and accuracy.
          </p>
        </div>
      </div>


      <!-- ----------- (3) CHEMOTAXIS --------------------------------------------------------------- -->
      <div class="research-item">
        <video id="myVideo" controls autoplay muted loop width="250px">
          <source src = "videos/Brownian Motion Colony Simulation.mp4" type="video/mp4">
        </video>

        <div class="research-description">
          <h3 class="papertitle">Stochastic modelling and simulation of bacterial chemotaxis</h3>
          <p class = "justified-text">
            <i>Supervisor: Prof. Ritwik Kumar Layek, Dept. of Electronics & Electrical Communication Engineering, IIT Kharagpur</i> <br>
            The project uses MATLAB to simulate random bacterial movement in the presence/absence of chemical gradients with the focus on modelling the probabilistic nature of chemotaxis while incorporating directional bias that determine how bacteria move towards or away from chemical signals. Using stochastic processes like random walks, Monte Carlo simulations and Brownian motion, the project looks to explore the dynamic behavior of bacterial populations under varying environmental conditions.
          </p>
        </div>
      </div>

      <!-- ----------- (2) CTPR SPAIN PROJECT --------------------------------------------------------------- -->
      <div class="research-item">
        <a href="images/ctpr_spain_project_graphical_abstract.avif">
          <img src="images/ctpr_spain_project_graphical_abstract.avif" alt="CTPR Spain project Graphical abnstract" class="research-img">
        </a>

        <div class="research-description">
          <h3 class="papertitle">Design of Novel CTPR Variants via Tryptophan Substitutions to Enhance Electron Conductivity</h3>
          <p class = "justified-text">
            <i>Supervisor: Prof. Aitziber L. Cortajarena, Biomolecular Nanotechnology lab, Centro de Investigaci√≥n Cooperativa en Biomateriales CIC biomaGUNE</i> <br>
            This project involved the design of two novel Consensus tetratricopeptide repeat (CTPR) protein variants (in PyMOL) containing strategic tryptophan (W) substitutions along the convex and concave surfaces of the protein such that the models exhibit higher stability and enhanced electronic conductivity, making them promising candidates for the development of novel electroactive and conductive materials for biomedical applications.
          </p>
        </div>
      </div>


      <!-- ----------- (1) PROTEIN SPLICING --------------------------------------------------------------- -->
      <div class="research-item">
        <a href="images/Protein_splicing.avif">
          <img src="images/Protein_splicing.avif" alt="Protein Splicing" class="research-img">
        </a>

        <div class="research-description">
          <h3 class="papertitle">Designing therapeutics to target protein-protein interactions leading to diseases</h3>
          <p class = "justified-text">
            <i>Supervisor: Prof. Soumya De, Protein Engineering lab, Dept. of Bioscience & Biotechnology, IIT Kharagpur</i><br>
            This project involved the development of novel methods for the synthesis of cyclic peptides to inhibit cancer-causing interactions.
            I conducted extensive literature study on mechanisms of intein splicing, interaction of proteins with cyclic peptides and performed analysis of protein structures using PyMOL. I also learned important laboratory procedures in molecuar biology - cloning, SDS PAGE, PCR, site-directed mutagenesis, protein purification by Ni-NTA Affinity Chromatography, primary culture & protein expression.
          </p>
        </div>
      </div>

    </section>

    <footer>
      <p class="credit">
        Source code credit to <a href="https://github.com/jonbarron/website">Dr. Jon Barron</a>.
      </p>
    </footer>
  </div>
</body>

</html>
